python/chronos/src/bigdl/chronos/utils.py

```diff
@@ -30,3 +31,33 @@ def wrapped(*args, **kwargs):
             return function(*args, **kwargs)
         return wrapped
     return deprecated_decorator
+
+
+class LazyImport:
+    """
+        code adaptted from https://github.com/intel/neural-compressor
+        /blob/master/neural_compressor/utils/utility.py#L62
+
+        :param module_name: The name of module imported later
+        :param pkg: prefix path.
+    """
+    def __init__(self, module_name, pkg=None):
+        self.module_name = module_name
+        self.pkg = pkg
+
+    def __getattr__(self, name):
+        try:
+            module = importlib.import_module(self.module_name, package=self.pkg)
+            mod = getattr(module, name)
+        except:
+            spec = importlib.util.find_spec(str(self.module_name + '.' + name))
+            mod = importlib.util.module_from_spec(spec)
+            spec.loader.exec_module(mod)
+        return mod
+
+    def __call__(self, *args, **kwargs):
+        function_name = self.module_name.split('.')[-1]
+        module_name = self.module_name.split(f'.{function_name}')[0]
```

Comment:
I think you may just want to first check if there is a function name (by checking if there is a `.` in module_name) and then devide into 2 paths.
1. function_name is valid, we will follow the same code here.
2. function_name is not valid, we will just `return module(*args, **kwargs)`

Reply:
I don't check the function name judgment here, but I can add it later.
The purpose of splitting is to import the parent module. If the function name is invalid, then `__getattr__` will report an error.